sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -469,7 +478,42 @@ public void analyzeSentimentWithLanguage() {
                         sentenceSentiment.getConfidenceScores().getNegative());
                 }
             });
-        // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.analyzeSentiment#string-string
+        // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.analyzeSentiment#String-String
+    }
+
+    /**
+     * Code snippet for {@link TextAnalyticsAsyncClient#analyzeSentiment(String, String, AnalyzeSentimentOptions)}
+     */
+    public void analyzeSentimentWithLanguageWithOpinionMining() {
+        // BEGIN: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.analyzeSentiment#String-String-AnalyzeSentimentOptions
+        textAnalyticsAsyncClient.analyzeSentiment("The hotel was dark and unclean.", "en",
+            new AnalyzeSentimentOptions().setIncludeOpinionMining(true))
+            .subscribe(documentSentiment -> {
+                List<MinedOpinion> positiveMinedOpinions = new ArrayList<>();
+                List<MinedOpinion> mixedMinedOpinions = new ArrayList<>();
```

Comment:
unused?

Reply:
it should be neutral, not mixed. I will fix it.

Reply:
I am wrong. AspectSentiment and OpinionSentiment could have positive, negative, and mixed. So it should be keep.